src/plugin.js

```diff
@@ -240,7 +261,11 @@ class VR extends Plugin {
     }
 
     msgs.forEach((msg) => {
-      videojs.log(msg);
+      if (typeof msg === 'string') {
```

Comment:
why not always do `videojs.log('VR:', msg)`?

Reply:
BEATEN! 😛